<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Entrega</class>
 <widget class="QDialog" name="Entrega">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Definir Área de Entrega</string>
  </property>
  <widget class="QComboBox" name="comboBox_estados">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Cidade</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_cidade">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_procurabairro">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>721</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Entre com bairro ou rua&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_bairros">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>271</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_procurar">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>70</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Procurar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Estado</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_apagar">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>170</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Apagar
Seleção</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_entrega">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>551</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_inserirbairros">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>100</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Inserir
Bairros</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
